### PR TITLE
Add Yakuza/Like A Dragon bar ambience

### PR DESCRIPTION
**Pull request by ThomasEricB**
https://www.youtube.com/watch?v=Q0GtyZbHJDM

Reasoning: I think this collection of songs just "fit the mood" of your plugin. These songs were made as background ambience for the bars in the [Yakuza/Like A Dragon (in Japanese Ryū ga Gotoku) video game series](https://en.wikipedia.org/wiki/Yakuza_(franchise)).

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -23,6 +23,13 @@ const SOUNDSCAPES: Record<string, Soundscape> = {
 		isLiveVideo: true,
 		youtubeId: "jfKfPfyJRdk",
 	},
+	yakuzabar: {
```
Comment (andrewmcgivery) [inline main.ts:26]: Small nit pick... can you move this down with Animal Crossing and Nintendo? :)

File: main.ts [hunk 2]
```diff
@@ -23,6 +23,13 @@ const SOUNDSCAPES: Record<string, Soundscape> = {
...
+		id: "yakuzabar",
+		name: "Yakuza/Like a Dragon bar ambience",
+		nowPlayingText: "Yakuza bar ambience",
+		isLiveVideo: true,
```
Comment (andrewmcgivery) [inline main.ts:30]: This should be `false`

**Comment by andrewmcgivery** [comment]
@ThomasEricB thank you for the PR! Small changes requested but otherwise looks good. :)

**Comment by ThomasEricB** [comment]
@andrewmcgivery Thanks. Just committed the small changes

**Review by andrewmcgivery — APPROVED** [review]
Thanks so much for this contribution! Will target it for version 1.1 within the next week or so!

**Pull request merged**